site/content/how-to/traffic-management/tls-passthrough.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "TLS Passthrough"
+weight: 800
+toc: true
+docs: "DOCS-000"
+---
+
+Learn how to use TLSRoutes to configure TLS Passthrough load-balancing with NGINX Gateway Fabric.
+
+## Overview
+
+In this guide, we will show how to configure TLS passthrough for your application, using a [TLSRoute](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.TLSRoute).
+
```

Comment:
shouldn't we elaborate here more with a short overview of what we will do in the guide?

Reply:
I think this overview is sufficient because the guide is very simple. Open to suggestions for things to add here, though.

Reply:
It's a tight enough use case that no other explanation is necessary. 

The premise is self-evident: extra words become noise for the reader.